### DISCUSSION TITLE
[Feature Request] Let me toggle between total duration and remaining duration in Now Playing screen.

### DISCUSSION THREAD

**Discussion by bikram-agarwal** [PaulWoitaschek/Voice#1311] [Ideas]
On the audiobook playing screen, the left end of the seekbar shows the elapsed time and the right end shows the total time of the part being played. It would be nice if I toggle that to show time remaining to complete the part being played.

**Comment by PaulWoitaschek** [comment]
I'm generally open to that idea but I don't currently have a vision how to design that properly. Any suggestions / sketches welcome.

**Comment by bikram-agarwal** [comment]
I was thinking - just tap that time and it will toggle. i.e. say it is showing total runtime right now. I tap that and it will toggle to show the remaining runtime. I tap that again, and it'll toggle to show total runtime again.

**Comment by edcaron** [comment]
I think you could show both progress bars. The current file and the total of the book

**Comment by firecat53** [comment]
SmartAudioBook player has two attached parallel bars that are slightly different colors, one showing overall progress and one showing chapter/file progress. It's pretty intuitive.

**Comment by jtagcat** [comment]
> I was thinking - just tap that time and it will toggle. i.e. say it is showing total runtime right now. I tap that and it will toggle to show the remaining runtime. I tap that again, and it'll toggle to show total runtime again.

You could accidentally skip to the end of the chapter this way.



> SmartAudioBook player has two attached parallel bars that are slightly different colors, one showing overall progress and one showing chapter/file progress. It's pretty intuitive.

2 bars gets cluttered :/

**Comment by dessalines** [comment]
This would be really nice to have: a time remaining in the chapter. Either a setting, or tapping on the end time to show the time remaining.
